specs/app.spec.js

```diff
@@ -25,37 +25,25 @@ describe('When the user has csv files in the files folder', function () {
         });
 
         it('Should have inserted the t1 csv as expected', function (done) {
-            var connection = mysql.createConnection(DB_CONFIG);
-            connection.query('SELECT * FROM t1', (error, data) => {
-                connection.end();
-                if (error) {
-                    done(error);
-                }
-                expect(data).to.deep.equal([
-                    { '"id"': "1", 'first name': '"SMH"', "person's email": 'smh@email.com' },
-                    { '"id"': "2", 'first name': "MO'", "person's email": 'mo@email.com' },
-                    { '"id"': "3", 'first name': 'hello, world', "person's email": 'someemail@email.com' }
-                ]);
-                done();
-            });
+            connectAndQuery('SELECT * FROM t1')
```

Comment:
Try returning this. Remove done from the arrow function signature in the line above. Also remove the call to done() below. Also do the same for the rest.